src/test/addresses_mainnet.sol

```diff
@@ -0,0 +1,113 @@
+pragma solidity 0.5.12;
+
+contract Addresses {
+
+    mapping (bytes32 => address) public addr;
+
+    constructor() public {
+        addr["CHANGELOG"]           = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
+        addr["MULTICALL"]           = 0x5e227AD1969Ea493B43F840cfF78d08a6fc17796;
+        addr["FAUCET"]              = 0x0000000000000000000000000000000000000000;
```

Comment:
Should we just remove this since it's zero?

Reply:
I'm okay with leaving this just for consistency with the online changelog, but good either way.

Reply:
Yeah, this was lifted straight from the existing changelog.